redwood-cli/src/main/kotlin/app/cash/redwood/generator/composeGeneration.kt

```diff
@@ -228,14 +227,13 @@ sealed interface RowScope {
 
 internal object RowScopeImpl : RowScope
 */
-internal fun generateScopeAndScopedModifiers(schema: Schema, scope: KClass<*>): FileSpec {
+internal fun generateScope(schema: Schema, scope: KClass<*>): FileSpec {
   val scopeName = scope.simpleName!!
   val scopeType = ClassName(schema.composePackage(), scopeName)
   return FileSpec.builder(scopeType.packageName, scopeType.simpleName)
     .apply {
       val scopeBuilder = TypeSpec.interfaceBuilder(scopeType)
         .addAnnotation(Redwood.LayoutScopeMarker)
-        .addModifiers(SEALED)
```

Comment:
Removed the `sealed` keyword to make testing scoped functions easier. The scoped functions are still inaccessible unless users create their own scope instance in which case they're intentionally avoiding the DSL compiler safety.